mindee/documents/invoice.js

```diff
@@ -117,6 +117,7 @@ class Invoice extends Document {
   }
 
   #initFromApiPrediction(apiPrediction, pageNumber) {
+    this.words = []
```

Comment:
Si le this.words est dans la plupart les documents, il devrait être ajouté dans le constructeur de Document avec comme valeur par défaut `[]` afin d'éviter la duplication et l'appeler en passant simplement `apiPrediction["mvision"]` en nouveau paramètre quand il y en a besoin